packages/ketcher-core/src/application/render/raphaelRender.ts

```diff
@@ -214,7 +213,7 @@ export class Render {
 
       const isAutoScale = this.options.autoScale || this.options.downScale;
       if (!isAutoScale) {
-        const ext = Vec2.UNIT.scaled(scale);
+        const ext = Vec2.UNIT.scaled(12);
```

Comment:
ToDo: This value influences on distance between structure and canvas edge and should be made conditional.  If selection is moved, should have a small value (like 5 ) in order to structure always be near the edge of canvas. In other cases, should have 'scale' value = 40 as it was before, in order to structure has enough space before canvas edge for selection lasso, rotate controller and so on.